test/functional/zmq_test.py

```diff
@@ -6,29 +6,43 @@
 import configparser
 import os
 import struct
+import time
 
 from test_framework.test_framework import BitcoinTestFramework, SkipTest
-from test_framework.util import *
+from test_framework.util import (assert_equal,
+                                 bytes_to_hex_str,
+                                 )
+
+def wait_for_multipart(socket, timeout=60):
+    interval = 0.1
```

Comment:
Thanks! I didn't know about `zmq.RCVTIMEO`. Updated to use that instead of the timeout loop in https://github.com/bitcoin/bitcoin/pull/10555/commits/b1bac1cb7ea1560b46ff758864660a21a228ed67